modules/core/04-channel/keeper/events.go

```diff
@@ -189,7 +189,7 @@ func emitWriteAcknowledgementEvent(ctx sdk.Context, packet exported.PacketI, cha
 			sdk.NewAttribute(types.AttributeKeySrcChannel, packet.GetSourceChannel()),
 			sdk.NewAttribute(types.AttributeKeyDstPort, packet.GetDestPort()),
 			sdk.NewAttribute(types.AttributeKeyDstChannel, packet.GetDestChannel()),
-			sdk.NewAttribute(types.AttributeKeyAck, string(acknowledgement)), //nolint:staticcheck // DEPRECATED
+			sdk.NewAttribute(types.AttributeKeyAckHex, string(acknowledgement)), //nolint:staticcheck // DEPRECATED
```

Comment:
You have this attribute set twice with different values. It should be:
```suggestion
			sdk.NewAttribute(types.AttributeKeyAck, string(acknowledgement)), //nolint:staticcheck // DEPRECATED
```
I have not looked at other places but this broke some test in wasmd

Reply:
Yes, we will add it back. Sorry for the broken tests